### ISSUE TITLE
Version clash between refactor-nrepl and cider-nrepl

### ISSUE THREAD

**Issue by slipset** [clojure-emacs/refactor-nrepl#221]
The latest stable release of refactor-nrepl and cider-nrepl ships with different versions of tools.nrepl. 
This causes greif for any project with `:pedantic? :abort`

https://twitter.com/mtnygard/status/981963418566578176

**Comment by benedekfazekas** [comment]
true. meanwhile answer is same as is for clojure-emacs/clj-refactor.el#403: use latest snapshot, it is all good, unusually stable ;)

**Comment by MatthewDarling** [comment]
I think this is fixed as of #234, four days ago :) At least, with `:pedantic? :abort` in my `project.clj` and these plugins:

```clojure
:plugins [[refactor-nrepl "2.4.0"]
          [cider/cider-nrepl "0.18.0"]]
```

I'm able to start a REPL.

**Comment by bbatsov** [comment]
Yes, that's fixed now.
